modules/shared/src/main/scala/org/tessellation/kryo/KryoSerializer.scala

```diff
@@ -1,11 +1,14 @@
 package org.tessellation.kryo
 
-import cats.Applicative
 import cats.effect.{Async, Resource}
+import cats.implicits.catsSyntaxApplicativeErrorId
```

Comment:
Please not import anything from `cats.implicits`. We use `cats.syntax.___._` format for importing implicits